src/icons/fileIcons.ts

```diff
@@ -716,7 +716,8 @@ export const fileIcons: FileIcons = {
     { name: 'mxml', fileExtensions: ['mxml'] },
     { name: 'autohotkey', fileExtensions: ['ahk'] },
     { name: 'flash', fileExtensions: ['swf'] },
-    { name: 'swc', fileExtensions: ['swc'] },
+    { name: 'adobe-swc', fileExtensions: ['swc'] },
+    { name: 'swc', fileExtensions: ['swcrc'] },
```

Comment:
@coderabbitai Are there any other extensions for this type of file? If not, just write no and that's it